docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,891 @@
+Process Costing Library
+=======================
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. rubric:: IDAES Process Costing Framework
+
+This document outlines an example process costing library to support IDAES unit models. For a general overview of costing capabilities in IDAES, see the :ref:`IDAES Process Costing Framework<reference_guides/core/costing/costing_framework:IDAES Process Costing Framework>`.
+
+.. note:: Process costing methods are available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+The process costing module outlines capital cost methodology for generic IDAES unit operations assuming a 2018 cost basis.
```

Comment:
I think there needs to be a more general introduction here. I would start by introducing the source text and saying that this library implements some (all?) of the methods describe there and a bit of background on the assumptions (e.g. cost basis).